### ISSUE TITLE
procedure to migrate borg 1.x repo to borg 2.x

### ISSUE THREAD

**Issue by eoli3n** [borgbase/docs.borgbase.com#19]
I guess that the way to, is to:
- export the repo
- remove the repo on borgbase
- convert it locally with ``borg transfert``
- import the 2.0 repo

https://docs.borgbase.com/setup/import/
https://www.borgbackup.org/releases/borg-2.0.html

**Comment by m3nu** [comment]
Yes, I should try this. Can't it do a remote-to-remote transfer? Then one just needs to add a new v2 repo.

**Comment by eoli3n** [comment]
> Can't it do a remote-to-remote transfer?
If I tranfer remote to remote, eat double space for a short period of time, will I need to pay for the extra space ?

**Comment by m3nu** [comment]
Yes, but that's pretty neglible if it's only for a day. If it's just for a few hours it probably wont' get recorded at all.
